DFC.ServiceTaxonomy.ApiFunction/DFC.ServiceTaxonomy.ApiFunction.csproj

```diff
@@ -1,11 +1,22 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
     <AzureFunctionsVersion>v2</AzureFunctionsVersion>
   </PropertyGroup>
```

Comment:
We should treat warnings as errors...
`    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>`